### ISSUE TITLE
Highlighting issue after associated type with no default

### ISSUE THREAD

**Issue by natecook1000** [textmate/swift.tmbundle#24]
The following code sample shows a highlighting bug:

```swift
protocol Collection : Sequence {
    // highlighting stops from the end of the next line until the following '=':
    associatedtype IndexDistance : SignedInteger
    associatedtype Iterator : IteratorProtocol = IndexingIterator<Self>
}
```

The second `associatedtype` line should be highlighted like the first:

```swift
protocol Collection : Sequence {
    associatedtype IndexDistance : SignedInteger = Int
    associatedtype Iterator : IteratorProtocol = IndexingIterator<Self>
}
```

**Comment by jtbandes** [comment]
@natecook1000 I think this is already handled in #23, could you verify?

**Comment by natecook1000** [comment]
Yep, that's definitely the same behavior! Haven't had a chance to verify the fix.

**Comment by jtbandes** [comment]
I just merged #23, so this should be taken care of. Keep an eye out for whenever GitHub does another grammar update.
